mcs/swagger.yml

```diff
@@ -305,4 +349,48 @@ definitions:
         type: integer
         format: int64
         title: total number of groups
-
+  statement:
+    type: object
+    properties:
+      effect:
+        type: string
+      actions:
+        type: array
+        items:
+          type: string
+      resources:
+        type: array
+        items:
+          type: string
+  policy:
+    type: object
+    properties:
+      name:
+        type: string
+      version:
+        type: string
+      statements:
+        type: array
+        items:
+          $ref: "#/definitions/statement"
+  addPolicyRequest:
+    type: object
+    required:
+      - name
+    properties:
+      name:
+        type: string
+      definition:
+        type: string
+  listPoliciesResponse:
+    type: object
+    properties:
+      policies:
+        type: array
+        items:
+          $ref: "#/definitions/policy"
+        title: list of policies
+      total_policies:
```

Comment:
should this be just `total` to keep it consistent with the other apis?

Reply:
your PR yesterday for groups have `total_groups` but I agree, we should only use `total`